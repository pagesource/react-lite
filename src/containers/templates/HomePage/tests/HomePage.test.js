import React from 'react';
import { shallow } from 'enzyme';
import MockProvider from '../../../../../__mocks__/mockProvider';

import HomePage from '../index';

const editorialData = {
  title: 'Test Title',
  banner: 'Test Banner',
};

describe('HomePage component', () => {
  test('should render correctly', () => {
    const HomePageComponent = shallow(
      <MockProvider>
        <HomePage editorialData={editorialData} />
      </MockProvider>
    );
    expect(HomePageComponent).toMatchSnapshot();
  });
});
