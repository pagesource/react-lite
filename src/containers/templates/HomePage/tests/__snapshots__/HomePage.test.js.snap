// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component should render correctly 1`] = `
<Provider
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <MemoryRouter>
    <withRedux(withRouter(Connect(enhanced(HomePage))))
      editorialData={
        Object {
          "banner": "Test Banner",
          "title": "Test Title",
        }
      }
    />
  </MemoryRouter>
</Provider>
`;
